implementations/micrometer-registry-statsd/src/main/java/io/micrometer/statsd/StatsdMeterRegistry.java

```diff
@@ -79,8 +80,7 @@
     private final HierarchicalNameMapper nameMapper;
     private final Map<Meter.Id, StatsdPollable> pollableMeters = new ConcurrentHashMap<>();
     private final AtomicBoolean started = new AtomicBoolean();
-    DirectProcessor<String> processor = DirectProcessor.create();
-    FluxSink<String> fluxSink = new NoopFluxSink();
+    Sinks.Many<String> sink = new NoopManySink();
```

Comment:
As I was working on this, I wished there was a built-in no-op implementation, but then I wondered if we really need one. If we don't subscribe, then it is almost a no-op. Maybe our `NoopManySink` isn't really needed?

Reply:
Let's explore this in a separate change.